lib/extend/extendToRxn.js

```diff
@@ -18,10 +18,7 @@ function internalToRxn(reaction, rxnFct, options) {
   const rxn = rxnFct.call(reaction, programName);
   if (keepIdCode) {
     return rxn;
+  } else {
+    return rxn.replace(/^OCL_RXN_V1.0:.*$/m, '');
```

Comment:
Code is simpler indeed. I guess nobody would put as programName "OCL_RXN_V1.0:." ...